frontend/app/src/composables/api/balances/exchanges.ts

```diff
@@ -130,12 +130,13 @@ export const useExchangeApi = () => {
   const deleteExchangeData = async (
     name: SupportedExchange | '' = ''
   ): Promise<boolean> => {
-    const response = await api.instance.delete<ActionResult<boolean>>(
-      `/exchanges/data/${name}`,
-      {
-        validateStatus: validStatus
-      }
-    );
+    let url = `/exchanges/data`;
+    if (name) {
+      url += `/${name}`;
+    }
+    const response = await api.instance.delete<ActionResult<boolean>>(url, {
+      validateStatus: validStatus
+    });
```

Comment:
from the previous code;

when `name  = ''` 
then the url will be `/exchanges/data/`

else it will be `/exchanges/data/$name`

the new code is adding `/$name` if is it not `''`

was having a trailing slash on the url problematic? (`/exchanges/data/` vs `/exchanges/data`)

Reply:
yes the trailing slash is problematic, it throw 404 with trailing slash